### PR TITLE
feat: Add Tx Command-Line for Deactivate Deal

### PR DESCRIPTION
**Pull request by inchori**
Add Tx Command Line for Deactivate Deal.

### CODE REVIEW

File: x/market/client/cli/txDeal.go [hunk 1]
```diff
@@ -70,6 +71,35 @@ func CmdSellData() *cobra.Command {
...
 
+func CmdDeactivateDeal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deactivate-deal [dealId]",
+		Short: "deactivate-deal",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return nil
```
Comment (gyuguen) [inline x/market/client/cli/txDeal.go:82]:
```suggestion
				return err
```
It seems that the error return is missing.

File: x/market/client/cli/txDeal.go [hunk 2]
```diff
@@ -70,6 +71,35 @@ func CmdSellData() *cobra.Command {
...
+			}
+
+			dealId, err := strconv.ParseUint(args[0], 10, 64)
+			if err != nil {
+				return nil
```
Comment (gyuguen) [inline x/market/client/cli/txDeal.go:87]:
```suggestion
				return err
```
It seems that the error return is missing.

**Review by hansol-medi — APPROVED** [review]
LGTM

**Review by gyuguen — APPROVED** [review]
LGTM 👍

**Review by youngjoon-lee — COMMENTED** [review]
ACK

**Pull request merged**